pages/dkp/kaptain/1.3.0/user-management/index.md

```diff
@@ -32,20 +34,33 @@ Predefined `ClusterRoles` for Kubeflow tenants are as follows:
 For a chart of the permissions granted to each `ClusterRole`, see the [Permissions Charts](#permissions-charts).
 
 ## Onboarding new users
+
 ### Overview
+
 Kaptain provides an abstraction called `Profile` which is assigned to a user and bound to a namespace along with additional
 resources such as namespace-scoped service accounts, RBAC `RoleBinding`s, Istio `ServiceRole` and `ServiceRoleBinding`.
 Kubeflow grants users with namespace admin permissions for their namespaces.
 
 ### Automatic profile creation
-When an authenticated user logs into the system and visits the central dashboard for the first time, they trigger a profile creation automatically, this is referred to as a "Registration Flow".
-Automatic profile creation is disabled by default. To enable it, set the `registrationFlow` parameter to `true`:
 
-```
-kubectl kudo install --instance kaptain --namespace kubeflow --create-namespace ./kubeflow-1.4.0_1.3.0.tgz -p registrationFlow=true
-```
+When an authenticated user logs into the system and visits the central dashboard for the first time, they trigger a profile creation. This is referred to as a "Registration Flow". However, automatic profile creation is disabled by default.
+
+-   To **enable** it, ensure all [`kudo plans`][kudoplan] have been completed, and then set the `registrationFlow` parameter to `true`:
```

Comment:
not sure if we want to mention this but the exact command to run is
<img width="987" alt="image" src="https://user-images.githubusercontent.com/35975404/165968217-e436697e-75ca-4579-8537-499283e026c5.png">
and can be found [on this page](https://docs.d2iq.com/dkp/kaptain/1.3.0/install/konvoy-dkp/).